pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExporter.java

```diff
@@ -1048,7 +1045,7 @@ public int exportDataSetAsCsv(StudyDto studyDto, List<ActivityExtract> activitie
                         row.addAll(formatter.emptyRow());
                     } else {
                         if (instances.size() > 1) {
-                            LOG.error("[export] participant {} has {} instances of activity {} {}, will only export the latest one",
+                            LOG.warn("[export] participant {} has {} instances of activity {} {}, will only export the latest one",
```

Comment:
Turned these into `warn`s since they're not too useful and it's clogging up our alert channels.